bpfd/src/rpc.rs

```diff
@@ -345,13 +361,30 @@ impl Loader for BpfdLoader {
                             }
                         };
 
+                        // Map UUID to String for response
+                        let map_owner_uuid = match r.map_owner_uuid {
+                            Some(uuid) => uuid.to_string(),
+                            None => String::new(),
+                        };
+                        // Map Vec<UUID> to Vec<String> for response
+                        let mut map_used_by: Vec<String> = vec![];
+                        if let Some(uuid_list) = r.map_used_by.clone() {
```

Comment:
nit in the future you should be able to do something along the lines of 

```rust
let map_used_by: Vec<String> = r.map_used_by.unwrap_or(vec![])
    .iter()
    .map(|u| u.to_string())
    .collect();
```

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=11dd8153b73657b7ae15b2319fd637b5